src/rust/engine/src/nodes.rs

```diff
@@ -1018,11 +1018,11 @@ impl Node for NodeKey {
 
     let (started_workunit_id, user_facing_name) = {
       let user_facing_name = self.user_facing_name();
-      let higher_priority = context.session.should_handle_workunits() && user_facing_name.is_some();
-      let level = if higher_priority {
-        Level::Info
-      } else {
-        Level::Debug
+      let level = match self {
```

Comment:
Should probably move this into a method of `NodeKey`.

Reply:
Maybe, but I'm less sure. I think this depends on how in general we want to tune workunit levels. Note that the levels for the workunits associated with processes aren't set here, but rather in `process_execution/src/lib.rs` where those workunits themselves are created.

Reply:
I'm pretty sure it should. This method is long enough already, and the "level" of a node should be a self-contained property of the node?